lib/lightning_web/controllers/webhooks_controller.ex

```diff
@@ -6,33 +6,44 @@ defmodule LightningWeb.WebhooksController do
   # this gets hit when someone asks to run a workflow by API
   @spec create(Plug.Conn.t(), %{path: binary()}) :: Plug.Conn.t()
   def create(conn, %{"path" => path}) do
-    path
-    |> Enum.join("/")
-    |> Workflows.get_edge_by_webhook()
-    |> case do
-      nil ->
-        put_status(conn, :not_found)
-        |> json(%{})
-
-      %Workflows.Edge{target_job: %Jobs.Job{enabled: false}} ->
-        put_status(conn, :forbidden)
-        |> json(%{
-          message:
-            "Unable to process request, trigger is disabled. Enable it on OpenFn to allow requests to this endpoint."
-        })
-
-      edge ->
-        {:ok, %{work_order: work_order, attempt_run: attempt_run}} =
-          WorkOrderService.create_webhook_workorder(edge, conn.body_params)
-
-        resp = %{
-          work_order_id: work_order.id,
-          run_id: attempt_run.run_id,
-          attempt_id: attempt_run.attempt_id
-        }
-
-        conn
-        |> json(resp)
-    end
+    source_trigger_id = path |> List.last()
+
+    :telemetry.span(
+      [:lightning, :workorder, :webhook],
```

Comment:
It feels like it may be useful to also pull the workflow_id for this request as that may be more useful? The problem is that, if I am reading the code correctly, I can only get that from the edge (if I reuse the existing code). However, that would mean that I would move  a potentially expensive operation outside the span, where it would not be measured.